crates/bevy_render/src/mesh/mesh/conversions.rs

```diff
@@ -25,9 +25,21 @@
 //! ```
 
 use crate::mesh::VertexAttributeValues;
+use bevy_math::{IVec2, IVec3, IVec4, UVec2, UVec3, UVec4, Vec2, Vec3, Vec4};
 use bevy_utils::EnumVariantMeta;
 use thiserror::Error;
 
+macro_rules! convert_vec_of_vec {
```

Comment:
Why is this a macro instead of a function?

Reply:
Because the base types in `glam` are `XY`, `XYZ`, and `XYZW` instead of arrays, hence I can't use const generics to genericize over the size of the type. I could use a function, but then I would need three of them.